Sources/NostrSDK/Events/NostrEvent.swift

```diff
@@ -141,3 +141,47 @@ public class NostrEvent: Codable, Equatable, Hashable {
         tags.filter { $0.name == tag.rawValue }.map { $0.value }
     }
 }
+
+extension NostrEvent: MetadataCoding, RelayURLValidating {
+    private static let bech32NoteIdPrefix = "note"
+
+    /// Gets a bare `note`-prefixed bech32-formatted human-friendly id of this event, or `nil` if it could not be generated.
+    /// It is not meant to be used inside the standard NIP-01 event formats or inside the filters.
+    /// They are meant for human-friendlier display and input only.
+    /// Clients should still accept keys in both hex and npub format and convert internally.
+    ///
+    /// > Note: [NIP-19 bech32-encoded entities](https://github.com/nostr-protocol/nips/blob/master/19.md)
+    public var bech32NoteId: String? {
+        guard let data = id.hexDecoded else {
+            return nil
+        }
+        return Bech32.encode(NostrEvent.bech32NoteIdPrefix, baseEightData: data)
```

Comment:
Rather than hard-coding a second source of this information, why not use `Bech32IdentifierType.note.rawValue`?
```suggestion
        return Bech32.encode(Bech32IdentifierType.note.rawValue, baseEightData: data)
```

Reply:
You're right, I missed it. Nice catch!